src/IssueCreator/Main.cs

```diff
@@ -239,14 +244,33 @@ private void MoveItem(ListBoxWithSearch source, ListBoxWithSearch dest)
             {
                 dest.Items.Add(source.SelectedItem);
                 source.Items.Remove(source.SelectedItem);
+                if(dest == lstSelectedTags)
+                {
+                    s_model.Labels.Add(source.SelectedItem as string);
+                }
+                else
+                {
+                    s_model.Labels.Remove(source.SelectedItem as string);
+                }
```

Comment:
I am not sure I understand what this does :).

Reply:
It's a hacky data binding between `s_model.Labels` and `lstSelectedTags.Items` 😄

Reply:
Ah -- I see now. We are either adding or removing the label that was just being manipulated in the UI from the underlying model. Got it.